internal/pkg/cli/env_init.go

```diff
@@ -172,46 +189,137 @@ func (o *initEnvOpts) Execute() error {
 	return nil
 }
 
+func (o *initEnvOpts) validateCustomizedResources() error {
+	var vpcImport, vpcConfig bool
+	if (o.ImportVPC.ID != "") || (o.ImportVPC.PublicSubnetIDs != nil) || (o.ImportVPC.PrivateSubnetIDs != nil) {
+		vpcImport = true
+	}
+	if (o.AdjustVPC.CIDR.String() != deploy.EmptyIPNetString) || (o.AdjustVPC.PublicSubnetCIDRs != nil) || (o.AdjustVPC.PrivateSubnetCIDRs != nil) {
+		vpcConfig = true
+	}
+	if vpcImport && vpcConfig {
+		return errors.New("couldn't specify both vpc importing flags and vpc configuring flags")
+	}
+	if (vpcImport || vpcConfig) && o.UseDefault {
+		return errors.New("couldn't import or configure resources if use default flag is set")
+	}
+	return nil
+}
+
+func (o *initEnvOpts) askEnvName() error {
+	if o.Name != "" {
+		return nil
+	}
+
+	envName, err := o.prompt.Get(envInitNamePrompt, envInitNameHelpPrompt, validateEnvironmentName)
+	if err != nil {
+		return fmt.Errorf("prompt to get environment name: %w", err)
+	}
+	o.Name = envName
+	return nil
+}
+
+func (o *initEnvOpts) askEnvProfile() error {
+	// TODO: add this behavior to selector pkg.
+	if o.Profile != "" {
+		return nil
+	}
+
+	names := o.profileConfig.Names()
+	if len(names) == 0 {
+		return errNamedProfilesNotFound
+	}
+
+	profile, err := o.prompt.SelectOne(
+		fmt.Sprintf(fmtEnvInitProfilePrompt, color.HighlightUserInput(o.Name)),
+		envInitProfileHelpPrompt,
+		names)
+	if err != nil {
+		return fmt.Errorf("get the profile name: %w", err)
+	}
+	o.Profile = profile
+	return nil
+}
+
+func (o *initEnvOpts) askCustomizedResources() error {
```

Comment:
nit: `askToCustomizeVPC`

Reply:
Same reason as [this](https://github.com/aws/copilot-cli/pull/1232#discussion_r465996861)